translations/src/config.js

```diff
@@ -68,18 +69,53 @@ class Config {
     return Object.keys(this.supportedLanguages)
   }
 
-  getRTLLanguages (): string[] {
-    return entries(this.supportedLanguages)
-      .filter(([_, languageConfig]) => languageConfig.rtl)
-      .map(([languageCode, _]) => languageCode)
+  getSupportedLanguage (languageCode: string): LanguageType | void {
+    const fallbacks = this.fallbacks[languageCode]
+
+    if (fallbacks) {
+      const found = fallbacks.find(fallback => !!this.supportedLanguages[fallback])
```

Comment:
So basically this just checks that there are not fallbacks specified which are not supported? Shouldn't we rather ensure (e.g. with tests or flow types) that fallbacks are always supported?

Reply:
Also, why are there (possibly) more than one fallback for one language?

Reply:
i18next can work with multiple fallbacks. That means if a string is not availabel then it will look through all languages in that array.

Reply:
Yes this checks whether the requestsed language code is is a fallback of a supported language. That way it is also supported.

Reply:
Yeah but I think all language codes should only have one fallback. i.e. `de-si` should not have a fallback of `de` and `de-de`

Reply:
yes thats how it is configured atm. This is just how the config in translations js was defined. We don't have to use the full potential.

Reply:
That way we can simply pass it to i18next